content/dstu2/diagnostic-report.md

```diff
@@ -0,0 +1,38 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 API
+---
+
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Search
+
+Search for DiagnosticReports that meet supplied query parameters:
+
+    GET /DiagnosticReport?:parameters
+
+### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`patient`|`string`| The identity of a patient to list reports for. Example: `12345`
+`patient._id`|`string`| The identity of a patient to list reports for. Example: `12345`
+`subject:Patient`|`string`| The identity of a patient to list reports for. Example: `12345`
```

Comment:
I believe this should just be 'subject', based on what we have in the conformance.